packages/roosterjs-editor-plugins/lib/plugins/TableCellSelection/TableCellSelection.ts

```diff
@@ -127,4 +134,18 @@ export default class TableCellSelection implements EditorPlugin {
             editor.select(selection.table, null);
         }
     }
+
+    private handleInputEvent(editor: IEditor) {
+        // If typing on any cell selection, clear the first cell
+        const range = editor.getSelectionRangeEx();
+        if (range.type == SelectionRangeTypes.TableSelection) {
+            const row = range.ranges[0];
+            const firstCell = row.startContainer.childNodes[row.startOffset];
+            firstCell.childNodes.forEach(node => {
+                node.remove();
+            });
+        }
+        // Add undo snapshot after content deletion
+        editor.addUndoSnapshot();
+    }
```

Comment:
This will also trigger if type out of table. 

The problem is, we should not always add undo snapshot in that case. You can try in an empty doc, type something, then undo. Now each undo only remove one character, which is wrong.